pom.xml

```diff
@@ -100,13 +100,13 @@
       <groupId>org.eclipse.aether</groupId>
       <artifactId>aether-api</artifactId>
       <version>${aetherVersion}</version>
-      <scope>compile</scope>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.eclipse.aether</groupId>
       <artifactId>aether-util</artifactId>
       <version>${aetherVersion}</version>
-      <scope>compile</scope>
+      <scope>compile</scope> <!-- To work in Maven versions older than 3.9.0 -->
```

Comment:
Interesting, why?

Reply:
Maven 3.2.5 core extensions:
https://github.com/apache/maven/blob/2226900a49396ae834b749ff22c126aed89dbf5a/maven-core/src/main/resources/META-INF/maven/extension.xml#L152-L154

Does NOT enlist util, hence, if provided, it will result in CNFEx

Reply:
It is fixed in 3.9.0 and master... would not bother with anything in 3.8.x line anymore